pkg/apisix/cluster.go

```diff
@@ -167,6 +169,16 @@ func (c *cluster) syncCacheOnce() (bool, error) {
 		log.Errorf("failed to list ssl in APISIX: %s", err)
 		return false, err
 	}
+	streamRoutes, err := c.streamRoute.List(context.TODO())
```

Comment:
Seems to be a bugfix.

Reply:
Yes, it was missing but it's OK since stream route is not in use now.